packages/components/src/components/text/text.tsx

```diff
@@ -1,9 +1,21 @@
 import classNames from 'classnames';
 import React from 'react';
-import PropTypes from 'prop-types';
 import { isEmptyObject } from '@deriv/shared';
+import { TChildren, TStyle } from '../types';
 
-const Text = ({ children, size, color, align, weight, line_height, as, className, styles, ...props }) => {
+export type TTextProps = {
+    line_height?: string;
+    size?: string;
+    children?: TChildren;
+    color?: string;
+    align?: string;
+    weight?: string;
+    as?: string;
+    className?: string;
+    styles?: TStyle;
+};
+
+const Text = ({ children, size, color, align, weight, line_height, as, className, styles, ...props }: TTextProps) => {
     const class_styles = {
         '--text-size': `var(--text-size-${size || 's'})`,
```

Comment:
isn't it better to add these values as default values to the props instead of repeating the `||` statement?
i know that it was like this, but in this case we are improving the component performance.

Reply:
@ali-hosseini-fs, OK I'll update them